scipy/stats/stats.py

```diff
@@ -82,6 +82,9 @@
     obrientransform
     signaltonoise
     sem
+    zmap
+    zscore
+    iqr
```

Comment:
this is done now. deleting the comment to make it not show up in the PR conversation